assets/src/application/ui/display/icons/svgs/infinity.svg

```diff
@@ -0,0 +1,11 @@
+<svg
```

Comment:
source: https://iconmonstr.com/infinity/
licence: https://iconmonstr.com/license/

Reply:
it was tricky to align and position the text infinity text sign, this is why I opted for an icon

Reply:
This needs to be removed. We can't use this due to its license restrictions:

> 3. Prohibited Uses
> Licensee may not sell, redistribute, sublicense, share or otherwise transfer the Work to other people or entities.